types/genesis.go

```diff
@@ -28,7 +28,7 @@ type GenesisDoc struct {
 	ConsensusParams *ConsensusParams   `json:"consensus_params,omitempty"`
 	Validators      []GenesisValidator `json:"validators"`
 	AppHash         cmn.HexBytes       `json:"app_hash"`
-	AppOptions      interface{}        `json:"app_options,omitempty"`
+	AppStateJSON    json.RawMessage    `json:"app_options,omitempty"`
```

Comment:
We should be consistent with the Go name and JSON name

Reply:
k, Also just renamed to `AppState` unless there is a reason to include JSON in the name? Also what's with `omitempty`? I left it in there